doc/pathing.md

```diff
@@ -3,10 +3,10 @@ various utility functions. Where appropriate, edge distances may be passed in as
 matrix of real number values. The matrix should be indexed by `[src, dst]` (see [Getting Started](gettingstarted.html) for more information).
 
 ### Graph Traversal
-`Graph traversal` refers to a process that traverses vertices of a graph following certain order (starting from user-input sources). This package implements three traversal schemes:
-` `BreadthFirst`,
-` `DepthFirst`, and
-` `MaximumAdjacency`.
+"Graph traversal" refers to a process that traverses vertices of a graph following certain order (starting from user-input sources). This package implements three traversal schemes:
```

Comment:
Ugh. This was global search and replace. Could you change this to *Graph traversal*? Terms are emphasized.